web/src/hotkey.js

```diff
@@ -121,51 +121,144 @@ const keydown_either_mappings = {
-    122: {name: "zoom_to_message_near", message_view_only: true}, // 'z'
+    "*": {name: "star_deprecated", message_view_only: true}, // '*'
+    "+": {name: "thumbs_up_emoji", message_view_only: true}, // '+'
+    "=": {name: "upvote_first_emoji", message_view_only: true}, // '='
+    "-": {name: "toggle_message_collapse", message_view_only: true}, // '-'
+    "/": {name: "search", message_view_only: false}, // '/'
+    ":": {name: "toggle_reactions_popover", message_view_only: true}, // ':'
+    ">": {name: "compose_quote_reply", message_view_only: true}, // '>'
+    "?": {name: "show_shortcuts", message_view_only: false}, // '?'
+    "@": {name: "compose_reply_with_mention", message_view_only: true}, // '@'
+    A: {name: "stream_cycle_backward", message_view_only: true}, // 'A'
+    C: {name: "C_deprecated", message_view_only: true}, // 'C'
+    D: {name: "stream_cycle_forward", message_view_only: true}, // 'D'
+    G: {name: "G_end", message_view_only: true}, // 'G'
+    J: {name: "vim_page_down", message_view_only: true}, // 'J'
+    K: {name: "vim_page_up", message_view_only: true}, // 'K'
+    M: {name: "toggle_topic_mute", message_view_only: true}, // 'M'
+    P: {name: "narrow_private", message_view_only: true}, // 'P'
+    R: {name: "respond_to_author", message_view_only: true}, // 'R'
+    S: {name: "narrow_by_topic", message_view_only: true}, // 'S'
+    U: {name: "mark_unread", message_view_only: true}, // 'U'
+    V: {name: "view_selected_stream", message_view_only: false}, // 'V'
+    a: {name: "all_messages", message_view_only: true}, // 'a'
+    c: {name: "compose", message_view_only: true}, // 'c'
+    d: {name: "open_drafts", message_view_only: true}, // 'd'
+    e: {name: "edit_message", message_view_only: true}, // 'e'
+    g: {name: "gear_menu", message_view_only: true}, // 'g'
+    h: {name: "vim_left", message_view_only: true}, // 'h'
+    i: {name: "message_actions", message_view_only: true}, // 'i'
+    j: {name: "vim_down", message_view_only: true}, // 'j'
+    k: {name: "vim_up", message_view_only: true}, // 'k'
+    l: {name: "vim_right", message_view_only: true}, // 'l'
+    m: {name: "move_message", message_view_only: true}, // 'm'
+    n: {name: "n_key", message_view_only: false}, // 'n'
+    p: {name: "p_key", message_view_only: false}, // 'p'
+    q: {name: "query_streams", message_view_only: true}, // 'q'
+    r: {name: "reply_message", message_view_only: true}, // 'r'
+    s: {name: "narrow_by_recipient", message_view_only: true}, // 's'
+    t: {name: "open_recent_topics", message_view_only: true}, // 't'
+    u: {name: "show_sender_info", message_view_only: true}, // 'u'
+    v: {name: "show_lightbox", message_view_only: true}, // 'v'
+    w: {name: "query_users", message_view_only: true}, // 'w'
+    x: {name: "compose_private_message", message_view_only: true}, // 'x'
+    z: {name: "zoom_to_message_near", message_view_only: true}, // 'z'
 };
```

Comment:
Is the migration to using `e.key` something we need to do for the keydown mappings as well? Also we can probably get rid of the comments at the end of these lines :)